.github/workflows/close_issue_on_pr_merge.yml

```
@@ -0,0 +1,30 @@
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Close linked issues
+      if: github.event.pull_request.merged == true
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:7:69: Double quote to prevent globbing and word splitting

(shellcheck)